sdk/include/reactos/stubs.h

```diff
@@ -1,6 +1,10 @@
 #define WIN32_NO_STATUS
 #include <windef.h>
 
+#ifndef PRIx64
```

Comment:
Can we just `#include <inttypes.h>` instead or manually defining this, or is that not possible at this point in the code?

Reply:
Likely possible, another variant would be to hardcode the preprocessor definition constant into spec2def itself. The latter is inflexible, your variant adds dependency that might not be lightweight. I believe this is the golden middle, but other opinions are welcome.

Reply:
If the header is supposed to be from the compiler, then VS 2010 might cause a problem here.
Personally, I'd be fine with this define here for the reasons provided.

Reply:
That constant seems to be used inside the generated code made by spec2def ?

Reply:
Yes. <https://github.com/reactos/reactos/blob/master/sdk/tools/spec2def/spec2def.c#L326> and <https://github.com/reactos/reactos/blob/master/sdk/tools/spec2def/spec2def.c#L380>.

Reply:
@learn-more This is not spec2def code that is missing it, but the generated code, which is built with our sdk, which has inttypes.h. I'd be fine with either.

Reply:
@ThFabba Do you consider the `#include` to be a blocker here?

Reply:
@ThFabba Do you consider the `#include` to be a blocker here?

Reply:
No, either solution seems acceptable. The include is still my personal preference, but it's far from being a blocker.